### ISSUE TITLE
New issues?

### ISSUE THREAD

**Issue by emanjavacas** [INL/BlackLab-server#7]
I've seen blacklab-server is now part of the blacklab repository. I imagine new issues should also go there?

**Comment by jan-niestadt** [comment]
Yes, that would be the best place for it, although I will also respond if someone posts here of course. :-)

**Comment by jan-niestadt** [comment]
Let's leave this issue open so others can see. I've updated the title to be a bit more descriptive.
